master/src/main/scala/org/apache/celeborn/service/deploy/master/quota/QuotaManager.scala

```diff
@@ -339,12 +350,14 @@ class QuotaManager(
     var nonExpired = used
     if (checkConsumptionExceeded(used, threshold)) {
       val sortedConsumption =
-        appMap.sortBy(_._2)(Ordering.by((r: ResourceConsumption) =>
-          (
-            r.diskBytesWritten,
-            r.diskFileCount,
-            r.hdfsBytesWritten,
-            r.hdfsFileCount)).reverse)
+        appMap.filter(app =>
+          appQuotaConfig.getOrDefault(app._1, defaultQuotaConfig).quotaInterruptShuffleEnabled)
```

Comment:
Seems that you only used the `quotaInterruptShuffleEnabled` in the expire application method, why not respect it in the `interruptShuffleEnabled` method.

Reply:
@FMX, `interruptShuffleEnabled` is used for server instead of application. When `interruptShuffleEnabled` is true, the `quotaInterruptShuffleEnabled` of application may be false for high priority.